app/components/Logo/logo.module.scss

```diff
@@ -2,6 +2,9 @@
 @use "styles/breakpoints";
 
 .logo {
+  width: 180px;
+  height: 50px;
+
   @include mixin.media-w(breakpoints.$laptop) {
     width: 96px;
```

Comment:
I guess that height of the logo should also be smaller on mobile, right?

Reply:
Add logo height for mobile. Thanks for the comments